packages/api-client/src/views/Settings/SettingsGeneral.vue

```diff
@@ -59,6 +59,16 @@ const getThemeColors = (
 const changeTheme = (themeId: ThemeId) => {
   workspaceMutators.edit(activeWorkspace.value?.uid ?? '', 'themeId', themeId)
 }
+
+const buttonStyles = cva({
+  base: 'w-full shadow-none text-c-1 justify-start pl-2 gap-2 border-1/2',
```

Comment:
So I would normally put all of my variants inside of ScalarButton as we shouldn't usually differ too much from the set button styles. BUT if this is a one off I guess its fine here. @hwkr what do you think?

+points for CVA.

Reply:
i guess i could come up with a better name too, but as we use these buttton styles in two components might make sense for a variant in the component button!

Reply:
love me some variants in the button component 😄

Reply:
I wonder if this should be a different component or an extension of the button since it's kind of a toggle / selectable button. This seems fine for now though